controllers/administration.js

```diff
@@ -1877,7 +1905,10 @@ router.all('/teams', function (req, res, next) {
 
 		const head = [
 			'Name',
-			'Klasse(n)',
+			'Mitglieder',
+			'Schule(n)',
+			'Erstellt am',
+			'',
 			''
```

Comment:
Ist die leere letzte Spalte in der Tabelle gewollt?

Reply:
ja, ist einmal für Icons die simbolsieren ob das team einen Owner hat oder das team Mitgliedern an anderen Schulen und für die Buttons die noch kommen. Dafür machen wir aber einen neuen PR damit zumindest mal das Anzeigen von Informationen drin ist